cmake/modules/OCGenerateResources.cmake

```diff
@@ -67,3 +77,30 @@ function(generate_legacy_icons theme_dir OUT)
     endforeach()
     set(${OUT} ${OWNCLOUD_ICONS} PARENT_SCOPE)
 endfunction()
+
+function(generate_resources QRC_PATH PREFIX FILES)
+    __write_qrc_file_header(${QRC_PATH} l10n)
```

Comment:
A general generate_resources  and a hardcoded l10n?

Reply:
I'm rewriting this file at the moment to eliminate the remaining duplicate code. I'm not satisfied with the way the icon files are handled at the moment, and have a better, more generic solution in mind which I implement at the moment. The functions I added use too many parameters...

I'll fix the hardcoded `l10n` value, thanks for the hint.